qa/os/src/test/java/org/elasticsearch/packaging/test/PackagingTestCase.java

```diff
@@ -686,30 +686,13 @@ public void verifySecurityAutoConfigured(Installation es) throws Exception {
             assertThat(sh.run(es.executables().keystoreTool + " list").stdout, Matchers.containsString("autoconfiguration.password_hash"));
             configLines = Files.readAllLines(es.config("elasticsearch.yml"));
         }
+        assertThat(configLines, hasItem("#----------------------- Security auto configuration start -----------------------"));
+        assertThat(configLines, hasItem("#----------------------- Security auto configuration end -------------------------"));
```

Comment:
Should we use something similar to PEM like BEGIN / END instead?

Reply:
I don't think we _have to_ , we're not trying to follow a standard

Reply:
I'm resolving, but feel free to re-open if you feel strongly about this Fabio

Reply:
I see that @lockewritesdocs also commented on the draft google doc suggesting use of `BEGIN SECURITY AUTO CONFIGURATION` and `END SECURITY AUTO CONFIGURATION`

I'd be fine with this suggestion.

Reply:
I took that as him applying your suggestion to the text, not a +1. As commented above I don’t believe there is a need to mimic PEM for any reason so it’s up to individual preference. No additional objections to change it since you have a strong opinion about it

Reply:
I care less about whether the text mimics the PEM format and more about a clear distinction between beginning and end. I think that `BEGIN` and `END` works well, and using all uppercase sets the text apart from the surrounding text. I'm 👍 to the suggestion.

Reply:
ack